commonknowledge/mapped
# Make political boundary names and counts less cluttered

## Description
This PR adds a switch in the map display settings to toggle the display of political boundary names and member counts
It also reduces the font size of political boundary names and counts

## Motivation and Context
Addresses issue [MAP-703](https://linear.app/commonknowledge/issue/MAP-703/make-countlabels-less-cluttered)

## How Can It Be Tested?
Download and run locally
Go to a map page
Toggle the switch for displaying political boundary names in the map display settings
Observe the smaller font size of the names and member counts
